src/Oro/ORM/Query/AST/Platform/Functions/Postgresql/Timestampdiff.php

```diff
@@ -101,7 +92,10 @@ protected function getDiffForMinute(Node $firstDateNode, Node $secondDateNode, S
      */
     protected function getDiffForHour(Node $firstDateNode, Node $secondDateNode, SqlWalker $sqlWalker)
     {
-        return $this->getDiffForSecond($firstDateNode, $secondDateNode, $sqlWalker) . ' / 3600';
+        return $this->getRoundedValue(
```

Comment:
Seems same issue with rounding as for minutes exists.
Possibly we need to use floor() function here to avoid this problem.